distribution/docs/src/main/resources/_contents/_apis/application-service-api-contents.adoc

```diff
@@ -0,0 +1,131 @@
+
+=== Application Service API
+
+The Application service has multiple interfaces which are exposed on to the OSGi runtime for other applications to use.
```

Comment:
"...on to the OSGI runtime..." sounds awkward and I'm not quite clear what it means.
On top of?